ibm-xforce-vulnerabilities/info.json

```
@@ -0,0 +1,521 @@
+          "description": "Specify the number of vulnerabilities to be returned. default value is 200. The limit must not be larger than 200",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "integer"
+        },
+        {
+          "title": "Skip",
+          "name": "skip",
+          "tooltip": "Specify the number of Vulnerabilities to be skipped while searching.",
+          "description": "Specify the number of Vulnerabilities to be skipped while searching.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "integer"
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "search_vulnerabilities",
+      "title": "Search Vulnerabilities",
+      "description": "Returns list of all vulnerabilities associated with the search term.",
+      "category": "investigation",
+      "annotation": "search_vulnerabilities",
+      "output_schema": {
+        "total_rows": "",
+        "bookmark": "",
+        "rows": [
+          {
+            "type": "",
+            "xfdbid": "",
+            "updateid": "",
+            "updated": "",
+            "variant": "",
+            "title": "",
+            "description": "",
+            "description_fmt": "",
+            "risk_level": "",
+            "CVSS3_Version": "",
+            "CVSS3_PrivilegesRequired": "",
+            "CVSS3_UserInteraction": "",
+            "CVSS3_Scope": "",
+            "access_vector": "",
+            "access_complexity": "",
+            "confidentiality_impact": "",
+            "integrity_impact": "",
+            "availability_impact": "",
+            "temporal_score": "",
+            "remediation_level": "",
+            "remedy": "",
+            "remedy_fmt": "",
+            "reported": "",
+            "tagname": "",
+            "stdcode": [
+              ""
+            ],
+            "platforms_affected": [
+              ""
+            ],
+            "exploitability": "",
+            "consequences": "",
+            "references": [
+              {
+                "link_target": "",
+                "link_name": "",
+                "description": ""
+              }
+            ],
+            "report_confidence": ""
+          }
+        ]
+      },
+      "parameters": [
+        {
+          "title": "Search Query",
+          "name": "q",
+          "tooltip": "Search term For example, Heartbleed.",
+          "description": "Search term For example, Heartbleed. ",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text"
+        },
+        {
+          "title": "Start Date",
+          "name": "startDate",
+          "tooltip": "Specify the start of the date range for searching. e.g. 2016-01-01T00:00:00Z. Must not be later than endDate. If not specified, the range will not be restricted.",
```

```suggestion
          "tooltip": "Specify the start of the date range for searching. Must not be later than endDate. If not specified, the range will not be restricted.",
```
example is not required as we are asking user to choose date from calender.